src/components/join-flow/email-step.jsx

```diff
@@ -13,9 +16,18 @@ class EmailStep extends React.Component {
         super(props);
         bindAll(this, [
             'handleValidSubmit',
+            'validateEmailIfPresent',
             'validateForm'
         ]);
     }
+    validateEmailIfPresent (email) {
+        if (!email) return null; // skip validation if email is blank; null indicates valid
+        const localResult = isEmail.validate(email);
```

Comment:
maybe rename?  this variable represents whether it is valid, right?  I think isValid would be a bit more readable.

Reply:
Done